### ISSUE TITLE
eslintrc as a separate file

### ISSUE THREAD

**Issue by goatslacker** [goatslacker/alt#216]
Ok it's a little ridiculous, maybe we should move it to a separate file like you suggested earlier :)

I think we should strip the one from Airbnb, diff it and see if we can contribute back any good rules.

https://github.com/airbnb/javascript/blob/master/linters/.eslintrc

**Comment by jdlehman** [comment]
These look to be the rules we have defined that airbnb's style guide does not specify. (I might have missed some, there were a lot to go through).

```js
{
    "consistent-this": [2, "this"],
    "func-style": [0, "expression"],
    "generator-star": [2, "end"],
    "max-nested-callbacks": [2, 3],
    "new-parens": 2,
    "no-array-constructor": 2,
    "no-inline-comments": 2,
    "no-lonely-if": 2,
    "no-mixed-spaces-and-tabs": [2, "always"],
    "no-multiple-empty-lines": [1, {
      "max": 2
    }],
    "no-ternary": 0,
    "operator-assignment": 0,
    "quote-props": [2, "as-needed"],
    "sort-vars": [0, {
      "ignoreCase": true
    }],
    "space-in-brackets": [0, "never",
      {
        "arraysInArrays": false,
        "arraysInObjects": false,
        "singleValue": false,
        "objectsInArrays": false,
        "objectsInObjects": false,
        "propertyName": false
      }
    ],
    "space-in-parens": [2, "never"],
    "space-unary-ops": [2,
      {
        "words": true,
        "nonwords": false
      }
    ],
    "wrap-regex": [2, "never"]
}
```

**Comment by jdlehman** [comment]
Resolved by PR #227.

See above comment for things we could potentially contribute back to airbnb's style guide.  Keep in mind that some of these rules might be the defaults in eslint.
